### PR TITLE
Use the new (async-ring supported) swagger-ui routes

### PR DESCRIPTION
**Pull request by ikitommi**
changes for https://github.com/metosin/ring-swagger/pull/118

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ## 1.2.0-SNAPSHOT
 
 * Fix help-for for some restructure methods [#275](https://github.com/metosin/compojure-api/pull/275) by [Nicolás Berger](https://github.com/nberger)
+* In `compojure.api.swagger`, the `swagger-ui` and `swagger-docs` now take options map with `uri` key instead of separate optional path & vararg opts.
```
Comment (miikka) [inline CHANGELOG.md]: This is a breaking change here as well.

Comment (ikitommi) [inline CHANGELOG.md]: true that! Should there be a old-format assertion break-gracefully?

Comment (miikka) [inline CHANGELOG.md]: That would be great

Comment (ikitommi) [inline CHANGELOG.md]: actually, the function arity has changed and the routes are invoked only at runtime -> without turning those into macros, there will be a runtime error. But most likely people see that immedeately, so should fail nice'ish. And these are not in the `sweet` api I guess 99% people are using.

Comment (ikitommi) [inline CHANGELOG.md]: ok to merge?

Comment (miikka) [inline CHANGELOG.md]: Sounds good, then.

Comment (ikitommi) [inline CHANGELOG.md]: Actually, the old consumed varargs, so there was a common 1-arity version. Now with assertion for the parameter type.

**Pull request merged**